pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>io.curity.identityserver.plugin</groupId>
     <artifactId>identityserver.plugins.authenticators.freja-eid</artifactId>
-    <version>3.1.2</version>
+    <version>3.1.3</version>
```

Comment:
Since this change adds new functionality, should the `minor` value be updated instead of the `patch` one (assuming we are using semantic versioning here)?